src/functions/color.js

```diff
@@ -99,10 +98,7 @@ defineFunction({
         argTypes: ["color"],
     },
     handler({parser, breakOnTokenText}, args) {
-        const color = args[0];
-        if (!color) {
-            throw new ParseError("\\color not followed by color");
-        }
+        const color = assertNodeType(args[0], "color-token");
```

Comment:
We could do a `check` here instead and then we'd be able to keep the error message from before.

Reply:
This check should never fail here due to the `argTypes: ["color"],` above. It would fail someplace in `ParseNode` instead.